### DISCUSSION TITLE
Get roles and permissions for specific guard ('api')

### DISCUSSION THREAD

**Discussion by davidreyg** [apiato/apiato#726] [Q&A] [answered]
By default, Apiato has seeders to insert role admin and permissions for both guards ('api' and 'web')
Therefore, when authenticating a user using api (ex: postman). How can i get permissions and roles only for 'api' guard?
Thanks!

**Comment by Mohammad-Alavi** [comment, marked as answer]
This Actions in not implemented by default. You can create a new Action and just return the results based on `api` and `web` guards (maybe passed in by the user in the request or just the current use guard) or you update `GetUserPermissionsAction`. 

There is an `activeGuard()` helper method that you can use to get the current logged in user guard.
